### PR TITLE
fix: labelAlign should work in Form.Item

### LINKED ISSUE #16067: Form.Item labelAlign does not work
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/k3y5254l73)

### Steps to reproduce
```jsx
<Form.Item label="label" labelAlign="right">
     item
</Form.Item>
```

### What is expected?
Label align to left

### What is actually happening?
Label still align to right.

| Environment | Info |
|---|---|
| antd | 3.16.3 |
| React | 16.8.6 |
| System | Window 10 |
| Browser | Chrome |

---
The third item  is what to be expected with `text-align: left;`

Also, it works on `<Form>` but the type definition and documentation also said  it is available on  `<Form.Item>`.

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

fix #16067 

### 💡 Solution

Check `labelAlign` in props of Form.Item.

### 📝 Changelog

Fix `labelAlign` not works on Form.Item.

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/form/__tests__/label.test.js [hunk 1]
```diff
@@ -222,4 +222,8 @@ describe('Form', () => {
         .getDOMNode(),
     ).toBe(document.activeElement);
   });
+
+  it('should `labelAlign` work in FormItem', () => {
+    expect(render(<Form.Item label="test" labelAlign="left" />)).toMatchSnapshot();
+  });
```
Comment (afc163) [inline components/form/__tests__/label.test.js]: 加个 Form 和 Form.Item 都指定 `labelAlign` 的用例。

Comment (yoyo837) [reply components/form/__tests__/label.test.js]: 好巧，我刚才也PR了一个

File: components/form/Form.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { ConfigConsumer, ConfigConsumerProps } from '../config-provider';
 import { ColProps } from '../grid/col';
 import { Omit, tuple } from '../_util/type';
 import warning from '../_util/warning';
-import FormItem from './FormItem';
+import FormItem, { LabelAlign } from './FormItem';
```
Comment (yoyo837) [inline components/form/Form.tsx]: `LabelAlign` 在哪里申明的？ 🤔

Comment (yoyo837) [reply components/form/Form.tsx]: Sorry, 看到了

**Review by yoyo837 — APPROVED** [review]

File: components/form/FormItem.tsx [hunk 3]
```diff
@@ -14,12 +14,14 @@ import { FormContext, FormContextProps } from './context';
 
 const ValidateStatuses = tuple('success', 'warning', 'error', 'validating', '');
 
+export type LabelAlign = 'left' | 'right';
```
Comment (afc163) [inline components/form/FormItem.tsx]: FormLabelAlign

**Pull request merged**